### ISSUE TITLE
Generate Node IDs

### ISSUE THREAD

**Issue by dasconnor** [ava-labs/avalanche-cli#421]
Allow the CLI to generate new NodeIDs that can be used to provision an avago node.

**Comment by charlie-ava** [comment]
For security concerns.  Should we not have avalanchego create the cert/key then upload it to secrets manager without having an individual create it manually.  I worry that once someone sets up a validator, they only have to wait until stake is on there then uses the cert/key they used to create the node for undesired purposes.   We might want to take the human aspect of this out.  But I guess that's more a security question.

**Comment by holisticode** [comment]
That certainly is a security question, but it needs to be answered before we start implementing this ticket, as the two options are completely different workflows.

The disadvantage of this approach however is, that it is not possible to create all configurations and required IDs prior to the node starting, which was the main goal of this effort as I understood. The NodeID is required for the AddValidator transaction, which is a prerequisite to have a subnet be deployed with all the nodes ready.

A slight variation of this option could be done by starting up all nodes, with the subnet nodes not being connected, and the subnet thus being unsafe, until all nodes have been created and started and their NodeIDs generated and uploaded, and the AddValidator transactions have run. This would not be a template for mainnet deployments though. And finally, this scenario would have to consider that the AddValidator transactions require a starting time stamp from which on to validate. This might be tricky to get right, or will require a good latency into the future, making the startup time until the subnet is ready to be used longer.

**Comment by dasconnor** [comment]
As per our Slack discussion, this functionality will not be added to Avalanche-CLI, so closing in favor of an alternative solution
